### ISSUE TITLE
accessibility: cannot click to select text without causing chromedash-feature to close

### ISSUE THREAD

**Issue by chrisosaurus** [GoogleChrome/chromium-dashboard#636]
The chromedash-feature widgets contain text which cannot be selected without closing the widget. This seems to only happen for clicks on text within the description (class="desc").

Many of our feature descriptions contain links-as-text or code snippets in their descriptions which cannot be selected.

A work around is to click the 'view on a standalone page' button, as the standalone page version does not have this same bug.

Reproduction:
1) open https://www.chromestatus.com/features/5144752345317376
2) notice that the text includes a link as text to a google groups page
3) attempt to click that link, feature widget closes
4) re-open feature
5) attempt to highlight that link text, feature widget closes

Expected:
3) link to google group should be a link
5) selecting text should not close widget
